src/org/zaproxy/zap/extension/ascanrulesBeta/SessionFixation.java

```diff
@@ -391,33 +391,28 @@ public void scan() {
 	        			String vulndesc=Constant.messages.getString("ascanbeta.sessionidaccessiblebyjavascript.desc");
 	        			String vulnsoln=Constant.messages.getString("ascanbeta.sessionidaccessiblebyjavascript.soln");
 
-	        			//and figure out the risk, depending on whether it is a login page	        
-	        			int risk = Alert.RISK_LOW;
 	        			if (loginUrl) {
 	        				extraInfo += ("\n" + Constant.messages.getString("ascanbeta.sessionidaccessiblebyjavascript.alert.extrainfo.loginpage"));
-	        				//login page, so higher risk
-	        				risk = Alert.RISK_MEDIUM;
-	        			} else {
-	        				//not a login page.. lower risk
-	        				risk = Alert.RISK_LOW;
 	        			}
 	        			
-	        			//call bingo with some extra info, indicating that the alert is 
-	        			//not specific to Session Fixation, but has its own title and description (etc)
-	        			//the alert here is "Session id accessible in Javascript", or words to that effect.
-	        			bingo(risk, Alert.CONFIDENCE_MEDIUM, vulnname, vulndesc, 
-	        					getBaseMsg().getRequestHeader().getURI().getURI(),
-	        					currentHtmlParameter.getName(),  attack, 
-	        					extraInfo, vulnsoln, getBaseMsg());
-	        					
-	        			if ( log.isDebugEnabled())  {
-	        				String logMessage = MessageFormat.format("A session identifier in [{0}] URL [{1}] {2} field: " +
-	        							"[{3}] may be accessible to JavaScript", 
-	        							getBaseMsg().getRequestHeader().getMethod(),  
-	        							getBaseMsg().getRequestHeader().getURI().getURI(), 
-	        							currentHtmlParameter.getType(), 
-	        							currentHtmlParameter.getName());
-	        				log.debug(logMessage);
+	        			if (loginUrl) {
```

Comment:
The `loginUrl` check could be moved to if of L386 (and could remove the if in L394).